src/pages/IDResolutionPage/IDResolveRedirectionPage.tsx

```diff
@@ -0,0 +1,105 @@
+import { CSSProperties, useCallback, useMemo } from 'react';
```

Comment:
@bilalesi 
Looks good. My only suggestion is to change the error message from "The redirection performance is suboptimal; further investigation is required, Please check again", to something like "The redirection feature is in beta version. We have taken note of this error and are working on improving the feature".

This is not mandatory though and very subjective.